app/views/config_reports/_metrics.html.erb

```diff
@@ -1,21 +1,33 @@
 <% @totaltime = metrics.delete('total') %>
 <% metrics.delete_if{ |k,v| v < 0.001 } %>
 <div class="row">
+  <div id="report-metrics-chart" class="col-md-4"></div>
```

Comment:
TBH I'm not sure how I feel about the wrapper.
I actually think you can move the whole erb file to React and compose the charts there instead of replacing them individually in the erb.

How much effort will it be to move the whole file to react?

Reply:
@amirfefer what do you think about it?